src/main/scala/io/citrine/lolo/transformers/standardizer/Standardization.scala

```diff
@@ -0,0 +1,108 @@
+package io.citrine.lolo.transformers.standardizer
+
+/** A package of the shift/scale required to standardize values to zero mean and unit variance. */
+case class Standardization(shift: Double, scale: Double) {
+  require(scale > 0 && scale < Double.PositiveInfinity)
+
+  def apply(x: Double): Double = (x - shift) / scale
+
+  def invert(x: Double): Double = x * scale + shift
+}
+
+object Standardization {
```

Comment:
Same helper functions as what used to be on the `Standardizer` companion object, but renamed.